src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,9 +1,270 @@
 import React, { Component } from 'react';
+import ProductPhoto from './ProductPhoto';
+import ProductSize from './ProductSize';
+import ProductBonus from './ProductBonus';
+import ShoppingBag from './ShoppingBag';
 import './ProductDetail.scss';
 
 class ProductDetail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      productInfoList: [],
+      imageNum: 0,
+      isSize: true,
+      isBonus: true,
+      isShoppingBag: true,
+      size: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/productData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productInfoList: data.results,
+        });
+      });
+  }
+
+  postProduct = () => {
+    const { productInfoList } = this.state;
+    fetch('http://10.58.7.79:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        product_id: productInfoList.product_id,
+      }),
+    });
+  };
+
+  changeSize = size => {
+    this.setState({ size: size }, this.hideSizePick());
+  };
```

Comment:
```suggestion
  changeSize = size => {
    this.setState({ size }, this.hideSizePick);
  };
```
- 객체의 키와 값에 들어오는 변수명이 같을 경우 축약 가능합니다
- 이 경우 함수의 호출을 넘겨주는게 아니라 함수 자체를 넘겨주는게 맞을거 같아요

Reply:
넵👍